addons/website_slides/views/website_slides_templates_lesson.xml

```diff
@@ -163,18 +163,18 @@
                     <ul t-if="aside_slide.slide_resource_ids or aside_slide.question_ids" class="list-group px-2 mb-1 list-unstyled">
                         <t t-foreach="aside_slide.slide_resource_ids" t-as="resource">
```

Comment:
we will have empty ```<li class="pl-4">``` (that may be shown depending on style), suggestions:

```suggestion
                        <t t-foreach="aside_slide.slide_resource_ids" t-as="resource" t-if="resource.resource_type == 'url'">
```
or:

```suggestion
                        <t t-foreach="aside_slide.slide_resource_ids.filtered(lambda r: r.resource_type == 'url')" t-as="resource">
```